.github/linters/install_chrome.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+set -ex
+apt-get -y update
+apt-get -y upgrade
+wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
+apt-get install ./google-chrome-stable_current_amd64.deb
```

Comment:
Looks like we need `-y` on this as well, to confirm installing extra packages.

```suggestion
apt-get -y install ./google-chrome-stable_current_amd64.deb
```